zingolib/src/lightclient/test_features.rs

```diff
@@ -33,28 +64,6 @@ impl LightClient {
         .map_err(ZingoLibError::CantReadWallet)
     }
 
-    /// Panics if the address, amount or memo conversion fails.
-    pub async fn propose_send_from_send_inputs(
-        &self,
-        raw_receivers: Vec<(&str, u64, Option<&str>)>,
-    ) -> Result<TransferProposal, ProposeSendError> {
-        let request = self
-            .transaction_request_from_send_inputs(raw_receivers)
-            .expect("should be able to create a transaction request as receivers are valid.");
-        self.propose_send(request).await
-    }
-
-    /// Panics if the address, amount or memo conversion fails.
-    pub async fn quick_send_from_send_inputs(
-        &self,
-        raw_receivers: Vec<(&str, u64, Option<&str>)>,
-    ) -> Result<NonEmpty<TxId>, QuickSendError> {
-        let request = self
-            .transaction_request_from_send_inputs(raw_receivers)
-            .expect("should be able to create a transaction request as receivers are valid.");
-        self.quick_send(request).await
-    }
-
```

Comment:
these two functions are necessary and im not sure why they are being deleted?
#1096 replaces send_from_send_inputs with quick_send_from_send_inputs throughout all integ tests to test zip317

Reply:
My bad.   I followed your suggestion and added _all_ necessary functionality to zingo-testutils/src/lightclient.rs